cse-runner-app/src/main/java/com/farao_community/farao/cse/runner/app/dichotomy/CseIdccShiftDispatcher.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.cse.runner.app.dichotomy;
+
+import com.farao_community.farao.dichotomy.api.exceptions.ShiftingException;
+import com.farao_community.farao.dichotomy.shift.ShiftDispatcher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static com.farao_community.farao.cse.runner.app.dichotomy.CseCountry.*;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class CseIdccShiftDispatcher implements ShiftDispatcher {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CseIdccShiftDispatcher.class);
+    private static final Set<String> BORDER_COUNTRIES = Set.of(FR.getEiCode(), CH.getEiCode(), AT.getEiCode(), SI.getEiCode());
```

Comment:
You can use the powsybl Country enum instead of declaring a new enum Country especially since it is already used in the rest of the project.